src/webui/www/private/scripts/prop-general.js

```diff
@@ -81,9 +84,9 @@ window.qBittorrent.PropGeneral = (function() {
             loadTorrentDataTimer = loadTorrentData.delay(5000);
             return;
         }
-        const url = new URI('api/v2/torrents/properties?hash=' + current_id);
+        const propertiesUrl = new URI('api/v2/torrents/properties?hash=' + current_id);
         new Request.JSON({
-            url: url,
+            url: propertiesUrl,
```

Comment:
This change doesn't seem related to the progress bar, please extract them to another commit.